pyvista/plotting/plotter.py

```diff
@@ -943,22 +944,22 @@ def scalar_bars(self):  # numpydoc ignore=RT01
         return self._scalar_bars
 
     @property
-    def _before_close_callback(self):
+    def _before_close_callback(self) -> None:
```

Comment:
This one might return something based on the return type of `__before_close_callback`?  I recommend removing this one unless you want a more complex type investigation.

Reply:
Indeed, I removed it.